### PR TITLE
Fix npm link step in "npm run start:dev" && "npm run build:console"

### PR DESCRIPTION
**Pull request by SecondDim**
# PULL REQUEST

## 說明 (Description)

npm version >= 7.x 時， npm link 會成為 Global link。所以在開發時重新 link 時，需要先 unlink 才能刪除 dist/。

## 貢獻種類 (Type of change)

- [x] Bug fix (除錯 non-breaking change which fixes an issue)
- [ ] New feature (增加新功能 non-breaking change which adds functionality)
- [ ] Breaking change (可能導致相容性問題 fix or feature that would cause existing functionality to not work as expected)
- [ ] Doc change (需要更新文件 this change requires a documentation update)

**測試環境 (Test Configuration)**:
* OS: mac
* NodeJS Version: 16.13.1
* NPM Version: 8.2.0
* Docker Version:

## 檢查清單 (Checklist):

- [x] 我的程式碼遵從此專案的規範 (My code follows the style guidelines of this project)
- [x] 我有對於自己的程式碼進行測試檢查 (I have performed a self-review of my own code)
- [x] 我有在程式碼中提供必要的註解 (I have commented my code, particularly in hard-to-understand areas)
- [x] 我有在文件中進行必要的更動 (I have made corresponding changes to the documentation)
- [x] 我的程式碼更動沒有顯著增加錯誤數量 (My changes generate no new warnings)
- [x] 我有新增必要的單元測試 (I have added tests that prove my fix is effective or that my feature works)
- [x] 我有檢查並更正程式碼錯誤的拼字 (I have checked my code and corrected any misspellings)

我已完成以上清單，並且同意遵守 [Code of Conduct](CODE_OF_CONDUCT.md)

I have completed the checklist and agree to abide by the [code of conduct](CODE_OF_CONDUCT.md).

- [x] 同意 (I consent)

### CODE REVIEW

File: src/console.ts [hunk 1]
```diff
@@ -34,7 +44,7 @@ if (npmBuild.status !== 0) { console.error(`${npmBuild.output[1]}`); console.err
  * npm link/
  */
 console.log('[*] exec spawnSync: npm link')
-const npmLink = spawnSync('npm', ['link'])
+const npmLink = spawnSync('npm', ['link', '--bin-links'])
```
Comment (SecondDim) [inline src/console.ts:47]: `--bin-links` 預設為 true，補上僅提供參考

File: src/console.ts [hunk 2]
```diff
@@ -10,6 +10,16 @@ console.log('[config.environment]', config.environment)
 console.log('[*] args: ', args)
 console.log('[*] build typescript project.')
 
+/**
+ * npm unlink -g
+ */
+console.log('[*] exec spawnSync: npm unlink -g')
+const npmUnlink = spawnSync('npm', ['unlink', '-g'])
+// console.log(`[=] stdout: ${npmUnlink.stdout}`)
+// console.log(`[x] stderr: ${npmUnlink.stderr}`)
```
Comment (kth-tw) [inline src/console.ts:19]: 這個不拿掉嗎

File: src/console.ts [hunk 3]
```diff
@@ -10,6 +10,16 @@ console.log('[config.environment]', config.environment)
 console.log('[*] args: ', args)
 console.log('[*] build typescript project.')
 
+/**
+ * npm unlink -g
+ */
+console.log('[*] exec spawnSync: npm unlink -g')
+const npmUnlink = spawnSync('npm', ['unlink', '-g'])
+// console.log(`[=] stdout: ${npmUnlink.stdout}`)
```
Comment (kth-tw) [inline src/console.ts:18]: 這個不拿掉嗎

File: src/console.ts (hunk 2 above)
Comment (kth-tw) [inline src/console.ts:19]: 這個不拿掉嗎？

Comment (SecondDim) [reply src/console.ts:19]: debug 能用

**Review by kth-tw — APPROVED** [review]

**Pull request merged**